Translations-iOS/zh-Hant.xcloc/Localized Contents/zh-Hant.xliff

```diff
@@ -1129,6 +1155,7 @@ You can give access to the rest of the files or the parent folder.</source>
       </trans-unit>
       <trans-unit id="To prevent a timeout on large operation, enable this setting. The result will always be true if enabled." xml:space="preserve">
         <source>To prevent a timeout on large operation, enable this setting. The result will always be true if enabled.</source>
+        <target state="translated">若要防止在耗時較長的動作上發生逾時問題，可以啟用此設定。如果啟用這個選項，result將永遠是true。</target>
```

Comment:
`result` is not a variable so it can be translated as well. Refers to the output the shortcut will produce.

Reply:
@pan93412 do you feel right to change that one?

Reply:
I am not sure what it means. What is the result the shortcut will produce?

Reply:
When you execute a shortcut it can produce some result usually in form of a String, a Boolean. For example the Keka shortcuts produce a Boolean result (true or false). So the result is the output of that shortcut.

Reply:
A basic example is 2 + 2 = 4, so the result is 4 🤓

Reply:
Okay, understand. I will change it later. Thank you!

Reply:
```suggestion
        <target state="translated">若要防止在耗時較長的動作上發生逾時問題，可以啟用此設定。如果啟用這個選項，輸出結果將永遠是true。</target>
```

@aonez How about this?